schedule_36C3.py

```diff
@@ -189,6 +189,22 @@ def main():
     # remove lighthing talk slot to fill with individual small events per lighthing talk
     #full_schedule.remove_event(id=10380)
 
+    # remove breaks from lightning talk schedule import
+    full_schedule.remove_event(guid='bca1ec84-e62d-528a-b254-68401ece6c7c')
+    full_schedule.remove_event(guid='cda64c9e-b230-589a-ace0-6beca2693eff')
+    full_schedule.remove_event(guid='f33dd7b7-99d6-574b-9282-26986b5a0ea0')
+
+    # remove talks starting before 9 am
```

Comment:
@saerdnaer Anything that I should know about in the 36C3 Schedule app?

Reply:
there was a daily event in the morning at about 6:00, which yielded to an empty view as you have to scroll down to actually see the talks.

Reply:
Yes, I saw the session and the whitespace. Still I would not artificially exclude any session for the sake of usability.